tools/js-sdk-release-tools/src/changelog/changelogGenerator.ts

```diff
@@ -159,21 +159,36 @@ export class Changelog {
     }
 }
 
+// todo: special rules for HLC convert to Modular, will use a more generic method to replace
+function getRenamedOperationFromToMap(from: TSExportedMetaData, to: TSExportedMetaData):  {[id:string]: string} {
```

Comment:
Could you have a try with pr: https://github.com/Azure/azure-sdk-for-js/pull/29770?

what is the current changelog?

Reply:
![image](https://github.com/Azure/azure-sdk-tools/assets/98569699/86d27870-3b33-4bff-b5e2-86f97195251b)
have code compile error in automaticGenerateChangeLogAndBumpVersion.ts with the latest commit